projects/redemption_configs/configs_specs/configs/specs/config_type.hpp

```diff
@@ -213,6 +213,12 @@ inline void config_type_definition(type_enumerations & e)
       .value("notify", "(Same thing as 'allow') ")
       .value("deny",   "User action will be rejected")
     ;
+
+    e.enumeration_list("ClientAddressSent", "For set clientAddress field when RDP module is created.")
+      .value("no_address", "send 0.0.0.0")
+      .value("proxy", "send proxy client address or target connexion")
+      .value("front", "send user client address of front connexion")
```

Comment:
- Use snake_case for key name
- Change description to:
"Client Address to send to target (in InfoPacket)"
- Add Capitals in first Letter of descriptions

Reply:
"ClientAddressSent" to "client_address_sent" so? Because  it's a enumeration and on the previous lines, the enumerations are written in camelcase

Reply:
oh wait, you are right, this is the enum, not the option name :)